openlibrary/plugins/openlibrary/js/index.js

```diff
@@ -1,3 +1,28 @@
+import 'jquery';
+import 'jquery-migrate';
+// Slick#1.6.0 is not on npm
+import '../../../../vendor/js/slick/slick-1.6.0.min.js';
+// npm jquery-ui@1.12.1 package does not match the one we have here, so for now we load from vendor
+import '../../../../vendor/js/jquery-ui/jquery-ui-1.12.1.min.js';
+// For dialog boxes (e.g. add to list)
+import '../../../../vendor/js/colorbox/1.5.14.js';
+// jquery-sparkline @ 1.4.3
+import '../../../../vendor/js/jquery-sparkline/jquery.sparkline.js';
+// jquery-show-password#1.0 not on npm, no longer getting worked on
+import '../../../../vendor/js/jquery-showpassword/jquery.showpassword.js';
+// jquery.form#2.36 not on npm, no longer getting worked on
+import '../../../../vendor/js/jquery-form/jquery.form.js';
+// jquery-validate#1.6 not on npm
+import '../../../../vendor/js/jquery-validate/jquery.validate.js';
+// jquery-autocomplete#1.1 with modified
+import '../../../../vendor/js/jquery-autocomplete/jquery.autocomplete-modified.js';
+// jquery-flot 0.7.0
+import '../../../../vendor/js/wmd/jquery.wmd.min.js'
```

Comment:
This fell into the flot section; Can you add a comment like you've done for the others?